src/main/java/io/github/sskorol/pages/LoginPage.java

```diff
@@ -16,10 +16,11 @@
     private final By loginStatusButton = By.name("login_status");
 
     @Step("Login with username: \"{userName}\", password: \"{password}\"")
-    public void login(final String userName, final String password) {
+    public SearchPage login(final String userName, final String password) {
         phantomType(usernameInput, userName);
         phantomType(passwordInput, password);
         phantomClick(signInButton);
+        return new SearchPage();
```

Comment:
as we discussed before, new objects creation has no effect in terms of elements locating...

Reply:
We did it this way because after login we should use methods from the Search Page, not from the Login Page.

Reply:
ok, then why don't you use `PageFactory`?